### PR TITLE
fix(publish): update versioncreated on kill/correct

### PR DESCRIPTION
**Pull request by petrjasek**
it creates a new version so versioncreated should get updated

### CODE REVIEW

File: server/apps/publish/content/correct.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def on_update(self, updates, original):
         ArchiveCropService().validate_multiple_crops(updates, original)
```
Comment (sivakuna-aap) [inline server/apps/publish/content/correct.py]: Looks like your superdesk-core is pointing to old SHA. ArchiveCropService has been moved to superdesk-core as CropService.

File: server/apps/publish/content/correct.py [hunk 2]
```diff
@@ -26,6 +26,7 @@ def on_update(self, updates, original):
...
         super().on_update(updates, original)
         updates[ITEM_OPERATION] = ITEM_CORRECT
+        updates['versioncreated'] = utcnow()
```
Comment (sivakuna-aap) [inline server/apps/publish/content/correct.py]: Can we move this to super class BasePublishService?

**Comment by ioanpocol** [comment]
PR https://github.com/superdesk/superdesk/pull/1408 has fixed the e2e tests problems.
Please rebase.

File: server/apps/publish/content/correct.py (hunk 2 above)
Comment (petrjasek) [inline server/apps/publish/content/correct.py]: I think this should not happen when you publish, but only when you kill and correct

File: server/apps/publish/content/correct.py (hunk 1 above)
Comment (petrjasek) [inline server/apps/publish/content/correct.py]: yep there was a different PR updating it, it's merged now

File: server/apps/publish/content/correct.py (hunk 2 above)
Comment (sivakuna-aap) [inline server/apps/publish/content/correct.py]: Any specific reason for not updating versioncreated on publish? I think there's no difference between publish, correct and kill operations.

**Comment by ioanpocol** [comment]
Done, without errors.

./features/steps/steps.py:1833:121: E501 line too long (122 > 120 characters)

/home/travis/build.sh: line 45: cd: server: No such file or directory

/home/travis/build.sh: line 45: cd: server: No such file or directory

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**